Update configuration

JuMastro opened the pull request:
* Add Travis configuraton included codeClimate & tests

* Add .gittatributes

* Add .editorconfig

* Add eslint

Add eslint and plugins (import, node, promise) + standard set
Add .eslintrc.js & .eslintignore

* Add package.json with inital configuration + .lock

* Add CHANGELOG.md

JuMastro committed:
* Update configuration

* Add Travis configuraton included codeClimate & tests

* Add .gittatributes

* Add .editorconfig

* Add eslint

Add eslint and plugins (import, node, promise) + standard set
Add .eslintrc.js & .eslintignore

* Add package.json with inital configuration + .lock

* Add CHANGELOG.md

JuMastro commented:
# TODO:

## Travis
- Should link to [travis.com](https://travis-ci.com)
- Should set env vars (NPM_API_KEY, NPM_EMAIL & CC_TEST_REPORTER_ID)

CC_TEST_REPORTER_ID can be found from "Analysis > Test coverage" from codeClimate project settings.

## CodeClimate
- Should link to [codeClimate](https://codeclimate.com)

![image](https://user-images.githubusercontent.com/25881035/50364328-e6035e80-056f-11e9-9eea-1420aba99542.png)
![image](https://user-images.githubusercontent.com/25881035/50364305-c835f980-056f-11e9-8a0d-ce166d4bdf62.png)
![image](https://user-images.githubusercontent.com/25881035/50364399-1ba84780-0570-11e9-96ac-1ee96a8c8d33.png)

JuMastro commented:
# Important
- Should add branch security for `master`, wait for review + travis validation + (codeClimate validation?!)

Mystilund commented on package.json:
```diff
@@ -0,0 +1,42 @@
+  "name": "cherry",
+  "version": "0.0.1",
+  "description": "A light framework to quickly create a web server",
+  "main": "./src/index.js",
```
Wrong, this file doesn't exist

JuMastro replied:
It's update to `./src/cherry.js`

Mystilund commented on .editorconfig:
```diff
@@ -0,0 +1,12 @@
+root = true
```
File to delete

JuMastro replied:
This file is now deleted

JuMastro committed:
Fix errors

(From Florian instructions)

* Add space char from 'Route.js' comment

* Fix variables errors
Replace 'middlewares' by 'routeMiddlewares' l.53
Replace 'middleware' by 'routeMiddlewares' l.55
